optaplanner-examples/src/main/resources/org/optaplanner/examples/batchscheduling/solver/batchSchedulingSolverConfig.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<solver>
+	<!--  <environmentMode>FULL_ASSERT</environmentMode>  -->
+	<solutionClass>org.optaplanner.examples.batchscheduling.domain.Schedule</solutionClass>
+	<entityClass>org.optaplanner.examples.batchscheduling.domain.Allocation</entityClass>
+	<entityClass>org.optaplanner.examples.batchscheduling.domain.AllocationPath</entityClass>
+	<scoreDirectorFactory>
+		<!-- <easyScoreCalculatorClass>org.optaplanner.examples.batchscheduling.solver.score.BatchSchedulingEasyScoreCalculator</easyScoreCalculatorClass> -->
+		<incrementalScoreCalculatorClass>org.optaplanner.examples.batchscheduling.solver.score.BatchSchedulingIncrementalScoreCalculator</incrementalScoreCalculatorClass>
+		<!--<initializingScoreTrend>ANY</initializingScoreTrend> -->
+		<!--<assertionScoreDirectorFactory> -->
+		<!--<easyScoreCalculatorClass>org.optaplanner.examples.batchscheduling.solver.score.BatchSchedulingEasyScoreCalculator</easyScoreCalculatorClass> -->
+		<!--</assertionScoreDirectorFactory>-->
+	</scoreDirectorFactory>
+	<termination>
+		<minutesSpentLimit>12</minutesSpentLimit>
+	</termination>
+	<constructionHeuristic>
+		<queuedEntityPlacer>
+			<entitySelector id="placerEntitySelector">
+				<cacheType>PHASE</cacheType>
+				<selectionOrder>ORIGINAL</selectionOrder>
+				<entityClass>org.optaplanner.examples.batchscheduling.domain.AllocationPath</entityClass>
+			</entitySelector>
+			<changeMoveSelector>
+				<entitySelector	mimicSelectorRef="placerEntitySelector"/>
+			</changeMoveSelector>
+		</queuedEntityPlacer>
+	</constructionHeuristic>
+	<constructionHeuristic>"src/main/resources/org/optaplanner/examples/batchscheduling/solver/batchSchedulingSolverConfig.xml"
+		<queuedEntityPlacer>
+			<entitySelector id="placerEntitySelector">
+				<cacheType>PHASE</cacheType>
+				<selectionOrder>ORIGINAL</selectionOrder>
+				<entityClass>org.optaplanner.examples.batchscheduling.domain.Allocation</entityClass>
+			</entitySelector>
+			<changeMoveSelector>
+				<entitySelector mimicSelectorRef="placerEntitySelector"/>
+			</changeMoveSelector>
+		</queuedEntityPlacer>
```

Comment:
We cannot rely on default constructionHeuristic (i.e. <constructionHeuristic/>) as there are multiple entity classes (i.e. AllocationPath and Allocation). Following error is thrown if default constructionHeuristic is used.

**Error:**
The config (QueuedEntityPlacerConfig(null, null)) has no entityClass configured and because there are multiple in the entityClassSet ([class org.optaplanner.examples.batchscheduling.domain.Allocation, class org.optaplanner.examples.batchscheduling.domain.AllocationPath]), it cannot be deduced automatically.

Reply:
Correct, my bad.